app.rb

```diff
@@ -0,0 +1,182 @@
+require 'date'
+
+require './book'
+require './student'
+require './rental'
+require './teacher'
+require './user_input'
+
+def main_choices
+  "Please choose an option by entering a number:
+1 - List all books
+2 - List all people
+3 - Create a person
+4 - Create a book
+5 - Create a rental
+6 - List all rentals for a given person
+7 - Exit"
+end
```

Comment:
- [ ] Please notice that the main point of creating the `main.rb` file is to divide the code and make the entry point to run the app starts from there by creating the UI choices in a `main` method that is identified in this file. Please refactor this method from the `app.rb` file.